crane/config.go

```diff
@@ -153,8 +155,10 @@ func NewConfig(location string, prefix string) Config {
 	config.initialize()
 	config.path = path.Dir(configFile)
 	config.prefix = prefix
+	config.tag = tag
 	milliseconds := time.Now().UnixNano() / 1000000
 	config.uniqueId = strconv.FormatInt(milliseconds, 10)
+	config.overrideWithGlobalFlags()
```

Comment:
I think since the tag handling is the only thing we do now, I would actually name this more specifically, so that it is clear it does something with tags.